.github/workflows/sentry_resque_test.yml

```diff
@@ -25,7 +25,12 @@ jobs:
         working-directory: sentry-resque
     name: Ruby ${{ matrix.ruby_version }}, options - ${{ toJson(matrix.options) }}
     runs-on: ubuntu-latest
+    env:
+      RUBYOPT: ${{ matrix.options.rubyopt }}
+      BUNDLE_GEMFILE: ${{ github.workspace }}/sentry-resque/Gemfile_with_rails.rb
```

Comment:
TODO: I think we can just move Rails related dependencies to `Gemfile` later

Reply:
@st0012 I removed it already and added a rails group to the Gemfile, then added `BUNDLE_WITHOUT="rails" bundle exec rake` to run tests w/o rails loaded. Seems to work well.